README.md

```diff
@@ -2,14 +2,16 @@
 
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/dotnet/interactive/master?urlpath=lab) [![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/interactive/interactive-ci?branchName=master)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=744&branchName=master)
 
-.NET interactive provides data scientists and developers a way to explore data, experiment with code, and try new ideas effortlessly. Use .NET Interactive to build .NET Jupyter notebooks or custom interactive coding experiences.
+.NET Interactive gives developers, data scientists, makers, and learners tools to write code and see the results immediately. You can explore data, experiment with code, and try new ideas. Use .NET Interactive to build .NET Jupyter notebooks or custom interactive coding experiences including bots and custom REPLs.
```

Comment:
This feels a little wordy.  How about this?
.NET Interactive is a group of tools and APIs that enables users to write/ learn code, explore data, and share code across multiple products and roles. 
Use .NET interactive to create interactive developers across notebooks, web, bots, and more. Looking forward to seeing what you do with it.

Reply:
I like it.